src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,14 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.length > 0 ? (
```

Comment:
you can convert to boolean like this:
```suggestion
    {!!comments.length ? (
```